dev-python/moviepy/moviepy-0.2.2.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Video editing with Python"
+HOMEPAGE="http://zulko.github.io/moviepy/"
+SRC_URI="https://github.com/Zulko/moviepy/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	"
+RDEPEND="
+	dev-python/decorator[${PYTHON_USEDEP}]
+	dev-python/imageio[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/tqdm[${PYTHON_USEDEP}]
+	"
```

Comment:
Missing virtual/ffmpeg rdep:
MoviePy depends on the software FFMPEG for video reading and writing: https://github.com/Zulko/moviepy/blob/master/docs/install.rst
https://github.com/Zulko/moviepy/blob/master/moviepy/video/io/ffmpeg_writer.py#L70